vite.preview.config.ts

```diff
@@ -22,13 +23,5 @@ export default defineConfig({
   },
   worker: {
     format: 'es',
-    plugins: [
-      replace({
```

Comment:
I have no idea why we need this plugin. Seems like remove it and it works as well.